src/main/java/org/zalando/problem/spring/web/advice/AdviceTrait.java

```diff
@@ -278,4 +278,10 @@ default void log(
         return entity;
     }
 
+    default ResponseEntity<Problem> process(
```

Comment:
Can you move this one before the original `process` method?

Reply:
Sure. Done.